.circleci/config.yml

```diff
@@ -2931,8 +2863,11 @@ jobs:
       runtime:
         type: string
         default: ci-test-executor
+      config_type:
+        type: string
       config_file:
         type: string
+        default: ""
```

Comment:
nit: this param, which is not used any more, could be removed.

Reply:
Leaving this parameter in place as this can be used , if necessary.